parkings/api/operator/urls.py

```diff
@@ -1,9 +1,25 @@
+from django.conf.urls import url
 from rest_framework.routers import DefaultRouter
 
 from ..url_utils import versioned_url
+from .enforcement_domain import EnforcementDomainView
 from .parking import OperatorAPIParkingViewSet
 
-router = DefaultRouter()
+
+class Router(DefaultRouter):
+    def get_urls(self):
+        urls = super().get_urls()
+        return urls + [
+            url(r"^enforcement_domain/$", EnforcementDomainView.as_view(), name="enforcement_domain")
```

Comment:
Use a ViewSet for this rather than a custom view. That would allow using a normal route.